WordPress/gutenberg
# Conditionally display filter reset button.

## What?
Only display the filter reset button when:

1. Filters are applied
2. The button is focussed

## Why?
When no filters are added the button adds noise to the UI. In terms of discoverability it's arguably more helpful to reveal the button when it's actionable.   

## How?
Adjust the opacity of the button when focus is within `.dataviews-filters__reset-button-container`.

## Testing Instructions
1. Open a data view like Manage all pages.
2. Observe no reset button.
3. Tab to the reset button and notice that it appears, disabled.
4. Add filters, notice the reset button appears and can be actioned.
5. Reset the filters and notice that focus remains on the disabled button, and that it remains visible until focus moves.


https://github.com/WordPress/gutenberg/assets/846565/de80ec7d-ee29-4b84-8d7d-b9976adfecb6
